itest/rust/src/object_tests/property_template_test.rs

```diff
@@ -0,0 +1,504 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+//! Testing that GDScript and rust produces the same property info for properties exported to Godot.
```

Comment:
Also here, it looks like there's a ton of code that could be generated. Otherwise, we have hundreds of lines to maintain manually.

Although admittedly, things like `#[init(default = Plane::new(Vector3::new(1.0,0.0,0.0), 0.0))]` make it a bit harder. I'm fine with skipping those until we found a more uniform way; see also #456.